CoffeeChat/Models/User.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  User.swift
+//  CoffeeChat
+//
+//  Created by Lucy Xu on 5/7/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import Foundation
+
+struct User: Codable {
+
+    let clubs: [String]
+    let firstName: String
+    let googleID: String
+    let graduationYear: String
+    let hometown: String
+    let interests: [String]
+    let lastName: String
+    let netID: String
+    let major: String
```

Comment:
... j k l m n o p ...

Reply:
lol thanks for the reminder on the alphabet